geoapps/inversion/components/factories/receiver_factory.py

```diff
@@ -57,7 +57,13 @@ def concrete_object(self):
         elif "tdem" in self.factory_type:
             from SimPEG.electromagnetics.time_domain import receivers
 
-            return receivers.PointMagneticFluxTimeDerivative
+            if self.params.data_units == "dB/dt (T/s)":
+                return receivers.PointMagneticFluxTimeDerivative
+            elif self.params.data_units == "B (T)":
+                return receivers.PointMagneticFluxDensity
```

Comment:
What about H:
![image](https://github.com/MiraGeoscience/geoapps/assets/81254271/41fa332d-7bdb-4240-9fa9-e73f34adf9f1)

Reply:
The next else returns a PointMagneticField

Reply:
The next else returns a PointMagneticField receiver...